private/pkg/git/gittest/repository.go

```diff
@@ -107,6 +108,8 @@ func (r *repository) PackRefs(t *testing.T) {
 }
 func (r *repository) ResetHard(t *testing.T, ref string) {
 	runInDir(t, r.runner, r.repoDir, "git", "reset", "--hard", ref)
+	// pause for a sec and let Git update some internal state
+	time.Sleep(1 * time.Second)
```

Comment:
With the random content, is this still necessary?

Reply:
Yes, it was racy :(